protocol-dashboard/src/hooks/useConnectAudiusProfile.ts

```diff
@@ -53,13 +58,18 @@ export const useConnectAudiusProfile = (wallet: string) => {
       env: env === 'production' ? 'production' : 'staging',
       successCallback: handleSuccess,
       errorCallback: (errorMessage: string) => {
-        // Error calllback
+        window.removeEventListener('message', receiveUserId)
         console.error(errorMessage)
       }
     })
   }
 
   const loginWithAudius = async () => {
+    window.removeEventListener('message', receiveUserId)
+    receiveUserHandlePromise = new Promise(resolve => {
```

Comment:
Sneaky

Reply:
Just double checking we can get back to a repeatable state if something fails/want to try again, if those are actual requirements

Reply:
yes we can!